rm-datasource/src/test/java/io/seata/rm/datasource/exec/UpdateExecutorTest.java

```diff
@@ -103,49 +95,149 @@ public static void init() {
+        MySQLUpdateRecognizer recognizer = new MySQLUpdateRecognizer(sql, asts.get(0));
+        updateExecutor = new UpdateExecutor(statementProxy, (statement, args) -> null, recognizer);
+
+        TableRecords beforeImage = updateExecutor.beforeImage();
+        TableRecords afterImage = updateExecutor.afterImage(beforeImage);
+        Assertions.assertNotNull(beforeImage);
+        Assertions.assertNotNull(afterImage);
+    }
+
+    @Test
+    public void testBeforeAndAfterImageWithTableSchemaQuoteAndTableNameQuote() throws SQLException {
+        String sql = "update `seata`.`table_update_executor_test` set name = 'WILL' where id = 1";
+        List<SQLStatement> asts = SQLUtils.parseStatements(sql, JdbcConstants.MYSQL);
+        MySQLUpdateRecognizer recognizer = new MySQLUpdateRecognizer(sql, asts.get(0));
+        updateExecutor = new UpdateExecutor(statementProxy, (statement, args) -> null, recognizer);
+
+        TableRecords beforeImage = updateExecutor.beforeImage();
+        TableRecords afterImage = updateExecutor.afterImage(beforeImage);
+        Assertions.assertNotNull(beforeImage);
+        Assertions.assertNotNull(afterImage);
+    }
+
+    @Test
+    public void testBeforeAndAfterImageWithColumnQuote() throws SQLException {
+        String sql = "update table_update_executor_test set `name` = 'WILL' where `id` = 1";
         List<SQLStatement> asts = SQLUtils.parseStatements(sql, JdbcConstants.MYSQL);
         MySQLUpdateRecognizer recognizer = new MySQLUpdateRecognizer(sql, asts.get(0));
         updateExecutor = new UpdateExecutor(statementProxy, (statement, args) -> null, recognizer);
+
         TableRecords beforeImage = updateExecutor.beforeImage();
+        TableRecords afterImage = updateExecutor.afterImage(beforeImage);
         Assertions.assertNotNull(beforeImage);
-        Assertions.assertNotNull(updateExecutor.afterImage(beforeImage));
+        Assertions.assertNotNull(afterImage);
     }
 
     @Test
-    public void testAfterImage() throws SQLException {
+    public void testBeforeAndAfterImageWithUpperColumn() throws SQLException {
+        String sql = "update table_update_executor_test set NAME = 'WILL' where ID = 1";
+        List<SQLStatement> asts = SQLUtils.parseStatements(sql, JdbcConstants.MYSQL);
+        MySQLUpdateRecognizer recognizer = new MySQLUpdateRecognizer(sql, asts.get(0));
+        updateExecutor = new UpdateExecutor(statementProxy, (statement, args) -> null, recognizer);
+
         TableRecords beforeImage = updateExecutor.beforeImage();
         TableRecords afterImage = updateExecutor.afterImage(beforeImage);
+        Assertions.assertNotNull(beforeImage);
         Assertions.assertNotNull(afterImage);
+    }
 
-        afterImage = updateExecutor.afterImage(new TableRecords());
+    @Test
+    public void testBeforeAndAfterImageWithTableAliasAndUpperColumn() throws SQLException {
+        String sql = "update table_update_executor_test t set t.NAME = 'WILL' where t.ID = 1";
```

Comment:
```suggestion
        String sql = "update table_update_executor_test t set t.NAME = 'WILL', t.UPDATED = `567` where t.ID = 1";
```
Use case for adding table aliases and capitalizing `ON UPDATE CURRENT_TIMESTAMP` column names
增加表别名与`ON UPDATE CURRENT_TIMESTAMP`列名大写的用例